cocos/2d/assets/ttf-font.ts

```diff
@@ -60,7 +64,7 @@ export class TTFFont extends Font {
         __nativeName__: string;
         ext: string;
         __isNative__: boolean;
-    } {
+        } {
```

Comment:
extra space

Reply:
It's automatically formated by while save according to `eslint intent` configuration.
![Screenshot 2024-08-05 at 16 48 03](https://github.com/user-attachments/assets/aee3d87a-7a6a-499d-84ad-2de9475e177e)

ESLint fixed that indentation
```ts
    @override
    get _nativeDep (): {
        uuid: string;
        __nativeName__: string;
        ext: string;
        __isNative__: boolean;
        } {
        return { uuid: this._uuid, __nativeName__: this._native, ext: path.extname(this._native), __isNative__: true };
    }
```